apps/modernization-ui/src/styles/global.scss

```diff
@@ -64,3 +63,11 @@ a {
         display: none;
     }
 }
+
+.h1 {
```

Comment:
question: Since these are global styles shouldn't they be applied at the element level and not through a class?  If there is a reason to use the class then these are already defined [here](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/components/heading/heading.module.scss).